### DISCUSSION TITLE
Calculating binding (free) energy

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#812] [Q&A]
Hi,

I am working with the adsorption of molecule A on molecule B. In order to study the interaction between them, I'm calculating the binding energy as E(bind) = E(AB) - E(A) - E(B).

I would like to know if there is a best way to calculate this binding energy.

Also, is it possible to calculate the binding free energy?

Thanks in advance,

Camps

**Comment by haneug** [comment]
This is a very fundamental question (for the "best way") that can not be answered so easily. I would recommend you to read this [article](https://onlinelibrary.wiley.com/doi/full/10.1002/ange.202205735) which deals with thermochemistry for molecules with DFT, but of course there are many more ways to calculate binding energies other than DFT.

**Reply by icamps** [reply]
Hi @haneug, 

Working now with a ligand/protein complex and xTB, this question back to my mind.

After looking the article you suggest, I arrived at equation (1) on the paper:

$$\Delta G = \Delta E + \Delta \delta G_{solv} + \Delta G_{mRRHO}$$

Here $\Delta E$ refers to the difference of the total electronic gas-phase energies, $\Delta \delta G_{solv}$ to the difference in solvation free energies and $\Delta G_{mRRHO}$ corresponds to the difference in the thermostatistical contributions.

Figure 7 of that paper, shows a workflow where CREST and CENSO software are including.

In the case of my system (a small molecule and a protein), it will be possible to use CREST/CENSO, efficiently, for such a big system with thousands of degree of freedom?

I optimized the three systems (isolated ligand, isolated protein, ligand/protein complex) with GFN2-xTB and did a molecular dynamic run with GFNFF. In all cases, used `gbsa` and `water` as solvent.

The question: from the results I have, it could be possible to make an approximation that allow me to calculate $\Delta G$ without considering the therm $\Delta G_{mRRHO}$?

**Reply by haneug** [reply]
Depending on your system size CREST and CENSO might be too expensive. The thermostatistical corrections can be obtained by a frequency calculation (which might be feasible on the GFN-FF level). You can of course neglect the thermostatistical contributions but especially in association reactions they can become quite large and relevant. I recommend the following article on this issue: [https://doi.org/10.1021/acs.jpclett.0c01930](https://doi.org/10.1021/acs.jpclett.0c01930). You might want to go with DFT single point energies on GFN2-xTB structures with GFN-FF thermostatistical corrections using a single point hessian (see here: [https://doi.org/10.1021/acs.jctc.0c01306](https://doi.org/10.1021/acs.jctc.0c01306).
